.goreleaser.yml

```diff
@@ -70,6 +76,23 @@ dockers:
     skip_push: false
     dockerfile: "docker/Dockerfile-gorelease"
 
+
+brews:
+  -
+    name: 
+    homepage: https://software.es.net/gdg
+    commit_msg_template: "Brew formula update for {{ .ProjectName }} version {{ .Tag }}"
+    description: Grafana Dash-n-Grab (GDG) -- Dashboard/DataSource Manager for grafana supporting backup/restore to local filesystem, s3, gcs, azure, and other S3 compatible storage engines. 
+    folder: Formula
+    repository:
+      owner: esnet
+      name: homebrew-gdg
+      token: "{{ .Env.HOMEBREW_TOKEN }}"
+      branch: main
+    commit_author:
```

Comment:
It would I think be nicer here if we had a more generic email vs personal email? This goes back to the convos in chat atm about what email addresses to use for things like this that we publish.

Reply:
Yeah, working on the ticket for that.  Keep in mind this is the commit email not the contact info.  That being said I think each project should be able to create some distribution list.